extras/nginx/Makefile.j2

```diff
@@ -9,7 +9,8 @@ KEY_PATH={{ KEY_PATH }}/{{ KEY_FILE }}
 KEY_SIZE={{ KEY_SIZE }}
 SERVER_URL={{ SERVER_URL }}
 SITES_AVAILABLE_DIR={{ SITES_AVAILABLE_DIR }}
-SITES_ENABLED_DIR= {{ SITES_ENABLED_DIR }}
+SITES_ENABLED_DIR={{ SITES_ENABLED_DIR }}
+SERVER_CFG_DIR={{ SITES_ENABLED_DIR }}/..
```

Comment:
Okay - I felt a bit dirty doing this ... but we are going to re-work all of this when it moved into Clank (that's what made me throw caution to wind :dash: 🎐)